### PR TITLE
Update star count

### PR DESCRIPTION
**Pull request by talentlessguy**
Some projects have earned some stars over time so this PR updates the list with more up-to-date stars

### CODE REVIEW

File: assets/json/fundable_oss.json [hunk 1]
```diff
@@ -10,15 +10,15 @@
     "id": 19,
     "name": "ffxiv-teamcraft/ffxiv-teamcraft",
     "url": "https://github.com/ffxiv-teamcraft/ffxiv-teamcraft",
-    "stars": 364,
-    "stars_display": "364"
+    "stars": 372,
+    "stars_display": "372"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:14]: Hi sorry for the late reply. I have request:

**Update:**
- ffxiv-teamcraft/ffxiv-teamcraft - 374 stars

File: assets/json/fundable_oss.json [hunk 2]
```diff
@@ -10,15 +10,15 @@
...
   },
   {
     "id": 20,
     "name": "bknds/ng-pi-admin",
     "url": "https://github.com/bknds/ng-pi-admin",
-    "stars": 132,
-    "stars_display": "132"
+    "stars": 133,
+    "stars_display": "133"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:21]: **Update:**
- bknds/ng-pi-admin - 135 stars

File: assets/json/fundable_oss.json [hunk 3]
```diff
@@ -66,43 +66,43 @@
...
+    "stars_display": "792"
   },
   {
     "id": 64,
     "name": "napi-rs/napi-rs",
     "url": "https://github.com/napi-rs/napi-rs",
-    "stars": 688,
-    "stars_display": "688"
+    "stars": 752,
+    "stars_display": "752"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:84]: **Update:**
- napi-rs/napi-rs - 759 stars

File: assets/json/fundable_oss.json [hunk 4]
```diff
@@ -66,43 +66,43 @@
...
   },
   {
     "id": 69,
     "name": "snapshot-labs/snapshot",
     "url": "https://github.com/snapshot-labs/snapshot",
-    "stars": 217,
-    "stars_display": "217"
+    "stars": 244,
+    "stars_display": "244"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:91]: **Update:**
- snapshot-labs/snapshot - 248 stars

File: assets/json/fundable_oss.json [hunk 5]
```diff
@@ -66,43 +66,43 @@
...
   },
   {
     "id": 70,
     "name": "vignette-project/vignette",
     "url": "https://github.com/vignette-project/vignette",
-    "stars": 136,
-    "stars_display": "136"
+    "stars": 146,
+    "stars_display": "146"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:98]: **Update:**
- vignette-project/vignette - 147 stars

File: assets/json/fundable_oss.json [hunk 6]
```diff
@@ -66,43 +66,43 @@
...
   },
   {
     "id": 71,
     "name": "lorcalhost/BTB-manager-telegram",
     "url": "https://github.com/lorcalhost/BTB-manager-telegram/",
-    "stars": 135,
-    "stars_display": "135"
+    "stars": 151,
+    "stars_display": "151"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:105]: **Update:**
- lorcalhost/BTB-manager-telegram -158 stars

File: assets/json/fundable_oss.json [hunk 7]
```diff
@@ -115,22 +115,22 @@
...
+    "stars_display": "289"
   },
   {
     "id": 105,
     "name": "ethereum-oasis/baseline",
     "url": "https://github.com/ethereum-oasis/baseline",
-    "stars": 503,
-    "stars_display": "503"
+    "stars": 512,
+    "stars_display": "512"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:126]: **Update:**
- ethereum-oasis/baseline - 513 stars

File: assets/json/fundable_oss.json [hunk 8]
```diff
@@ -115,22 +115,22 @@
...
   },
   {
     "id": 110,
     "name": "nuxt-community/typescript-template",
     "url": "https://github.com/nuxt-community/typescript-template",
-    "stars": 565,
-    "stars_display": "565"
+    "stars": 574,
+    "stars_display": "574"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:133]: **Update:**
- nuxt-community/typescript-template - 575 stars

File: assets/json/fundable_oss.json [hunk 9]
```diff
@@ -150,8 +150,8 @@
     "id": 135,
     "name": "eth-brownie/brownie",
     "url": "https://github.com/eth-brownie/brownie",
-    "stars": 709,
-    "stars_display": "709"
+    "stars": 821,
+    "stars_display": "821"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:154]: **Update:**
- eth-brownie/brownie - 832 stars

File: assets/json/fundable_oss.json [hunk 10]
```diff
@@ -164,22 +164,22 @@
     "id": 174,
     "name": "skick1234/DisTube",
     "url": "https://github.com/skick1234/DisTube",
-    "stars": 117,
-    "stars_display": "117"
+    "stars": 130,
+    "stars_display": "130"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:168]: **Update:**
- 132 stars

File: assets/json/fundable_oss.json [hunk 11]
```diff
@@ -164,22 +164,22 @@
...
   },
   {
     "id": 176,
     "name": "koxudaxi/fastapi-code-generator",
     "url": "https://github.com/koxudaxi/fastapi-code-generator",
-    "stars": 240,
-    "stars_display": "240"
+    "stars": 259,
+    "stars_display": "259"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:175]: **Update:**
- 263 stars

File: assets/json/fundable_oss.json [hunk 12]
```diff
@@ -164,22 +164,22 @@
...
   },
   {
     "id": 181,
     "name": "will-stone/browserosaurus",
     "url": "https://github.com/will-stone/browserosaurus",
-    "stars": 494,
-    "stars_display": "494"
+    "stars": 528,
+    "stars_display": "528"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:182]: **Update:**
- 530 stars

File: assets/json/fundable_oss.json [hunk 13]
```diff
@@ -192,12 +192,12 @@
     "id": 189,
     "name": "inherd/uncode",
     "url": "https://github.com/inherd/uncode/",
-    "stars": "175",
-    "stars_display": "175"
+    "stars": "179",
+    "stars_display": "179"
```
Comment (vinzvinci) [inline assets/json/fundable_oss.json:196]: **Update:**
- 182 stars

**Review by vinzvinci — CHANGES_REQUESTED** [review]
We can't expect an increase of GitHub stars in their project. But I'll be watching this PR, I'll merge your next changes.

**Comment by talentlessguy** [comment]
@vinzvinci wdym can't expect?

**Comment by vinzvinci** [comment]
> @vinzvinci wdym can't expect?

stars are increasing anytime on the repository, but don't worry about it. Is it okay if u change the things I requested? 

--- 
- I created issues related to it #51 (coz we are manually adding numbers of star)

**Comment by talentlessguy** [comment]
@vinzvinci yes it's ok

**Comment by talentlessguy** [comment]
@vinzvinci done, I've updated all the stars

**Comment by vinzvinci** [comment]
Hi @talentlessguy, thank you for your contributions, we decided not to merge PR for now, but your commits are still valid. Please look to other repositories `(there's a lot actually)`. I'll be closing this PR, please do understand.

**Reason:**
It depends on the developers of the sponsor's decision if they want to update their star.

**Comment by talentlessguy** [comment]
@vinzvinci wdym it depends on deve;oper of the sponsor's decision? those stars are not synced to github... and the JSON is static... so it's out of date

**Comment by vinzvinci** [comment]
This means we can't change it without their consent. I also like to merge this but we need to follow the rules. 

Yes, the JSON is static that's why I created issue #51. Don't worry there's a lot of repositories you can contribute here, we're still updating many things little by little.

**Pull request closed without merging**